src/Illuminate/Mail/Mailer.php

```diff
@@ -392,7 +390,11 @@ protected function sendSwiftMessage($message)
         }
 
         if (! $this->pretending) {
-            return $this->swift->send($message, $this->failedRecipients);
+            try {
+                return $this->swift->send($message, $this->failedRecipients);
+            } finally {
+                $this->swift->getTransport()->stop();
```

Comment:
@taylorotwell the problem is that `stop()` may throw an exception, but even if it'll be catched - it won't reset transport. You need to call either `reset()`, either `stop() > start()`.